compiler/literate/latex.ml

```diff
@@ -230,7 +230,7 @@ let rec law_structure_to_latex
          title=\\textcolor{black}{\\texttt{%s}},title after \
          break=\\textcolor{black}{\\texttt{%s}},before skip=1em, after \
          skip=1em]\n\
-         \\begin{minted}[numbersep=9mm, firstnumber=%d, \
+         \\begin{minted}[breaklines, numbersep=9mm, firstnumber=%d, \
```

Comment:
I'm actually not a fan of breaklines : the reason why I did not put it is to foster the programmer to keep lines short in the original program. Maybe the LaTeX literate programming backend can display an error when a line of code is too long to help the programmer avoid these lines going over the right margin ?

Reply:
I understand. Maybe one day there will be an autoformatter for Catala :thought_balloon:

Reply:
Yes there should be ! You can open an issue for that :)